doc/source/user_guide/style.ipynb

```diff
@@ -102,16 +117,110 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Table Styles\n",
+    "\n",
+    "Table styles allow you to control broader areas of the DataFrame styling with minimal HTML transfer. Much of the functionality of `Styler` uses individual HTML id tags to manipulate the output, which may be inefficient for very large tables. Using `table_styles` and otherwise avoiding using id tags can greatly reduce the rendered HTML.\n",
+    "\n",
+    "Table styles are also used to control features which can apply to the whole table at once such as greating a generic hover functionality:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s = Styler(df, cell_ids=False, uuid_len=1)\n",
+    "s.set_table_styles([{'selector': 'tr:hover',\n",
+    "                     'props': [('background-color', 'yellow')]}])\n",
+    "s.set_table_styles({\n",
+    "    'A': [{'selector': '',\n",
+    "           'props': [('color', 'red')]}],\n",
+    "    'B': [{'selector': 'td',\n",
+    "           'props': [('color', 'blue')]}]\n",
+    "}, axis=0, overwrite=False)\n",
+    "s.set_table_styles({\n",
+    "    3: [{'selector': 'td',\n",
+    "           'props': [('color', 'green')]}]\n",
+    "}, axis=1, overwrite=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The **advantage** of table styles is obviously the reduced HTML that it can create and the relative ease with which more general parts of the table can be quickly styled, e.g. by applying a generic hover, rather than having to apply a hover to each cell individually. Rows and columns as individual objects can only be styled in this way.\n",
+    "\n",
+    "The **disadvantage** of being restricted solely to table styles is that you have very limited ability to target and style individual cells based on dynamic criteria. For this one must use either of the other two methods. Also table level styles cannot be exported to Excel: to format cells for Excel output you must use the Styler Functions method below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Styler Functions\n",
+    "\n",
+    "Thirdly we can use the method to pass your style functions into one of the following methods:\n",
     "\n",
-    "When writing style functions, you take care of producing the CSS attribute / value pairs you want. Pandas matches those up with the CSS classes that identify each cell."
+    "- ``Styler.applymap``: elementwise\n",
+    "- ``Styler.apply``: column-/row-/table-wise\n",
+    "\n",
+    "Both of those methods take a function (and some other keyword arguments) and applies your function to the DataFrame in a certain way.\n",
+    "`Styler.applymap` works through the DataFrame elementwise.\n",
```

Comment:
not sure how easy this is to link to the actual docs for these but would be great if possible.

Reply:
i tried. i think it is possible but the relative path structure is quite complicated and i couldn't get it to work. never done it before so I have left out.